assets/javascript/game.js

```diff
@@ -1,219 +1,95 @@
 $(document).ready(function(){
 
-			var wins = 0;
-			var losses = 0;
-			
-			// This is the sum of the guesses added together. 
-			var counter = 0;
+  var audioElement1 = document.createElement('audio');
+  audioElement1.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV');
 
-			var crystal1 = 0;
-			var crystal2 = 0;
-			var crystal3 = 0;
-			var crystal4 = 0;
+  var audioElement2 = document.createElement('audio');
+  audioElement2.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3');
 
-			var audioElement1 = document.createElement("audio");
-      		audioElement1.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV");
+  var song = document.createElement('audio');
+  song.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3');
 
-      		var audioElement2 = document.createElement("audio");
-      		audioElement2.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3");
+  var wins = 0;
+  var losses = 0;
 
-      		var song = document.createElement("audio");
-      		song.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3");
+  var counter;
+  var numberGuess;
 
+  var crystal1 = $( '#crystal1_span' );
+  var crystal2 = $( '#crystal2_span' );
+  var crystal3 = $( '#crystal3_span' );
+  var crystal4 = $( '#crystal4_span' );
 
-			// This function chooses the random number for each crystal. 
-			function CrystalNumber(crystalName) {
-				return Math.ceil(Math.random() * 12 - 1) + 1;
-			};
 
-			crystal1 = CrystalNumber(crystal1);
-			crystal2 = CrystalNumber(crystal2);
-			crystal3 = CrystalNumber(crystal3);
-			crystal4 = CrystalNumber(crystal4);
-			console.log(crystal1, crystal2, crystal3, crystal4);
+  $('#startButton').on('click', function(){
+    audioElement1.play();
 
-			var numberGuess = Math.floor(Math.random() * 120) + 19;
+    $( '#startButton' ).effect( 'puff' );
+    $( '#crystal1_span' ).effect( 'pulsate' );
+    $( '#crystal2_span' ).effect( 'pulsate' );
+    $( '#crystal3_span' ).effect( 'pulsate' );
+    $( '#crystal4_span' ).effect( 'pulsate' );
+  });
 
-			// Check to see if the numberGuess is working. 
-			console.log(numberGuess);
 
-			// This sends the number the player is trying to reach to the #numberToGuess div.
-			$("#numberToGuess").html(numberGuess);
+  $('.crystal-btn').on('click', function(){
```

Comment:
Instead of writing out a listener for each individual crystal button, you can give them a generic class and then just refer to the button with `$(this)`. Helps to DRY up your code.